book/testing.rst

```diff
@@ -17,7 +17,7 @@ it has its own excellent `documentation`_.
 
 .. note::
 
-    Symfony works with PHPUnit 3.5.11 or later, though version 3.6.4 is
+    Symfony works with PHPUnit 3.5.11 or later, though version 4.2 is
```

Comment:
Actually, I'm not sure we can still run our testsuite on 3.5.11. I'm actually quite sure we started using 3.6+ features a long time ago, and we probably even use some newer features in a few place. So the requirement version should be bumped.

And the recommendation should be to use the latest stable version IMO (because this is what we actually use on Travis and on our development boxes, and so this is what actually works for sure)